### PR TITLE
Upgraded compiletest_rs to 0.9.0

### PR DESCRIPTION
**Pull request by oisyn**
The workspace inheritance is in there as well. I will rebase when those are merged in (done).

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -51,10 +51,4 @@ codegen-units = 256
...
 opt-level = 3
 incremental = true
-codegen-units = 256
-
-[patch.crates-io]
-# HACK(eddyb) needed to include this PR (until it's part of a release):
-# * Manishearth/compiletest-rs#249 (for `// only-*` support)
-# [shesp] The commit is part of 0.8 release. We can remove this once we upgrade `compiletest`.
-compiletest_rs = { git = "https://github.com/Manishearth/compiletest-rs.git", rev = "4ab843a1dc6ed9a82657d86d22397c6c5bb95b01" }
+codegen-units = 256
```
Comment (eddyb) [inline Cargo.toml:54]: You should e.g. "Files: Insert Final Newline" in VSCode, to fix this warning.

**Review by eddyb — CHANGES_REQUESTED** [review]
LGTM modulo the weird final newline thing.

**Comment by oisyn** [comment]
Wait why did android fail all of a sudden?

**Review by eddyb — APPROVED** [review]

**Comment by eddyb** [comment]
> Wait why did android fail all of a sudden?

* (I assume) blocked on #945

**Comment by oisyn** [comment]
> Wait why did android fail all of a sudden?

This is why: #945 🙂

**Pull request merged**